src/components/explore/AggregationsOverTimeGraph.svelte

```diff
@@ -143,6 +143,28 @@
     // Finally, set the flag to open the context menu.
     $showContextMenu = true; // eslint-disable-line no-unused-vars
   }
+
+  const getUniqueArray = (arr) => {
+    let a = arr.concat();
+    for (let i = 0; i < a.length; ++i) {
+      for (let j = i + 1; j < a.length; ++j) {
+        if (a[i] === a[j]) a.splice(j--, 1);
+      }
+    }
+    return a;
+  };
```

Comment:
This looks like something that could go in `src/utils/` and a test written for it. Or, I'm curious if https://lodash.com/docs/3.10.1#uniq accomplishes the same thing?